connector_dns/dns_view.xml

```diff
@@ -0,0 +1,162 @@
+                </tree>
+            </field>
+        </record>
+
+        <record id="dns_backend_form" model="ir.ui.view">
+            <field name="name">DNS Backend form</field>
+            <field name="model">dns.backend</field>
+            <field name="arch" type="xml">
+                <form>
+                    <header>
+                        <button name="button_connect" string="Connect" type="object" states="draft,exception"/>
+                        <button name="button_set_draft" string="Set to draft" type="object" states="done"/>
+                        <field name="state" widget="statusbar"/>
+                    </header>
+                    <sheet>
+                        <div class="oe_title" style="width: 390px;">
+                            <label class="oe_edit_only" for="name" string="Name"/>
+                            <h1>
+                                <field name="name" class="oe_inline"/>
+                            </h1>
+                        </div>
+                        <group>
+                            <field name="login" widget="email"/>
+                            <field name="password" password="True"/>
+                            <field name='version'/>
+                        </group>
+                    </sheet>
+                </form>
+            </field>
+        </record>
+
+        <!-- domain -->
+        <record model="ir.actions.act_window" id="dns_domain_action">
+            <field name="name">DNS Domain Action</field>
+            <field name="res_model">dns.domain</field>
+            <field name="view_type">form</field>
+            <field name="view_mode">tree,form</field>
+        </record>
+
+        <record id="dns_domain_tree" model="ir.ui.view">
+            <field name="name">DNS domain tree</field>
+            <field name="model">dns.domain</field>
+            <field name="arch" type="xml">
+                <tree>
+                    <field name="name"/>
+                    <field name="backend_id"/>
+                    <field name="state"/>
+                </tree>
+            </field>
+        </record>
+
+        <record id="dns_domain_form" model="ir.ui.view">
+            <field name="name">DNS domain form</field>
+            <field name="model">dns.domain</field>
+            <field name="arch" type="xml">
+                <form string='Domain'>
+                    <header>
+                        <button name="button_get_sub_domains" string="Get Sub domains" type="object" states="done"/>
+                        <field name="state" widget="statusbar"/>
+                    </header>
+                    <sheet>
+                        <div class="oe_title" style="width: 390px;">
+                            <label class="oe_edit_only" for="name" string="Name"/>
+                            <h1>
+                                <field name="name" class="oe_inline" attrs="{'readonly':[('state','!=','draft')]}"/>
```

Comment:
Wrong format, please fix it! e.g. `{'readonly': [('state', '!=', 'draft')]}`
Same for the rest of the file